tools/ncpms-api-crawler/ncpms-crawler.py

```
@@ -0,0 +1,162 @@
+
+    info = {"preventMethod": "", "ecologyInfo": "", "biologyPrvnbeMth": "", "chemicalPrvnbeMth": ""}
+    
+    if text.startswith("{"):
+        try:
+            data = json.loads(text)
+            svc = data.get("service", {})
+            info["ecologyInfo"] = remove_html_tags_from_api_text(svc.get("ecologyInfo", ""))
+            info["biologyPrvnbeMth"] = remove_html_tags_from_api_text(svc.get("biologyPrvnbeMth", ""))
+            info["chemicalPrvnbeMth"] = remove_html_tags_from_api_text(svc.get("chemicalPrvnbeMth", ""))
+            info["preventMethod"] = remove_html_tags_from_api_text(svc.get("preventMethod", ""))
+        except:
```

Ruff (0.15.10)

[error] Do not use bare `except`

(E722)